docs/protocol/index.md

```diff
@@ -2,133 +2,91 @@
+and tokens and assets in the network to come from different independent sources.
 
 Evmos enables these key features by:
 
-* Implementing Tendermint Core's Application Blockchain Interface ([ABCI](https://docs.tendermint.com/master/spec/abci/)) to manage the blockchain
 * Leveraging [modules](https://docs.cosmos.network/main/building-modules/intro.html) and other mechanisms implemented by the [Cosmos SDK](https://docs.cosmos.network/).
+* Implementing CometBFT's Application Blockchain Interface ([ABCI](https://docs.tendermint.com/master/spec/abci/)) to manage the blockchain.
 * Utilizing [`geth`](https://github.com/ethereum/go-ethereum) as a library to promote code reuse and improve maintainability.
-* Exposing a fully compatible Web3 [JSON-RPC](./apis/ethereum-JSON-RPC/JSON-RPC) layer for interacting with existing Ethereum clients and tooling Metamask, Remix, Truffle, etc).
-
+* Exposing a fully compatible Web3 [JSON-RPC](./apis/ethereum-JSON-RPC/JSON-RPC) layer for interacting with existing Ethereum clients and tooling (Metamask, Remix, Truffle, etc).
 
 The sum of these features allows developers to leverage existing Ethereum ecosystem tooling and
 software to seamlessly deploy smart contracts which interact with the rest of the Cosmos
-[ecosystem](https://cosmos.network/ecosystem)!
-
-import ProjectValue from '../../src/components/ProjectValue.js';
-
-
-## The EVMOS Token
-
-The denomination used for staking, governance and gas consumption on the EVM is the EVMOS. The EVMOS provides the utility of: securing the Proof-of-Stake chain, token used for governance proposals, distribution of fees to validator and users, and as a mean of gas for running smart contracts on the EVM.
+[ecosystem](https://cosmos.network/ecosystem).
 
-Evmos uses [Atto](https://en.wikipedia.org/wiki/Atto-) EVMOS as the base denomination to maintain parity with Ethereum.
+<!-- TODO ## High-level Systems Design (Add visualization) -->
 
-1 evmos = 10<sup>18</sup> aevmos
-
-This matches Ethereum denomination of:
+## Cosmos SDK
 
-1 ETH = 10<sup>18</sup> wei
+Evmos enables the full composability and modularity of the [Cosmos SDK](https://docs.cosmos.network/).
+As a Cosmos chain, Evmos is a sovereign blockchain with its own native token,
+that can connect to other chains through IBC. It includes standard modules from the Cosmos SDK,
+that work side to side with Evmos-specific modules, build by the Evmos core development team.
+Check out the [list of modules](modules/index.md) to get an overview of what each module is responsible for.
 
-## Cosmos Coins
+## CometBFT & ABCI
 
-Accounts can own Cosmos coins in their balance, which are used for operations with other Cosmos and transactions. Examples of these are using the coins for staking, IBC transfers, governance deposits and EVM.
+[CometBFT](https://github.com/cometbft/cometbft) (a fork of [Tendermint Core](https://docs.tendermint.com/)) consists of two chief technical components: a blockchain consensus
```

Comment:
As stated above, I think the information that Tendermint is now CometBFT is more fitting in the introductory section